TEST_WORKFLOW.md

```diff
@@ -0,0 +1,136 @@
+
+1.5 Partners and Developers implement this policy in good faith and in the context of sustainable, good software development practice.
+
+1.6 Projects may have their own workflow arrangements as far as these do not contradict this policy.
+
+## 2 Related documents
+
+2.1 Workflow policy is related to other X-Road tests development policy and regulations:
+- workflow policy implements Harmonized X-road tests document [Harmonized X-Road test environment](https://github.com/ria-ee/blob/master/HARMONIZED_TEST_ENVIRONMENT.md)
+
+## 3 Development model
+
+3.1	X-Road tests uses [Github](https://github.com/) and [Git](https://git-scm.com/) based version management.
+
+## 4	Repositories
+
+4.1	The following code and documentation repositories are used:
+- Master Test Repository - short name: X-Road-tests; hosted by: GitHub; managed by: RIA; purpose: release of X-Road automated tests; access: Head Architect has write access; read access: ALL.
+	- https://github.com/ria-ee/X-Road-tests
+
+4.2	Partners can establish their own, additional repositories, for backup, software distribution or other purposes.
+
+## 5	Branching pattern
+
+5.1	Branching pattern follows the [Gitflow model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). Two perpetual branches – `master` , `develop` and own `testfeature` branches.
+
+- `master` branch is used to release X-Road tests
+- `develop` branch is used to test development
+- `feature` branches are used to work on test features
+
+Main repository:
+- `master` branch https://github.com/ria-ee/X-Road-tests/tree/master
+	- releases of X-Road-tests that correspond to a specific release of X-Road source code. Several versions of X-Road-tests may point to the same X-Road source code version if necessary.
+	-  `master` branch releases are tagged with e.g "X-Road-tests for X-Road 6.16.0"
+- `develop` branch https://github.com/ria-ee/X-Road-tests/tree/develop
+	- Integration branch between Finland and Estonia (and other possible partners)
+	- Latest integrated test development source code 
+
+README.md file in the root of X-Road-tests must indicate the X-Road source code version for both MASTER and DEVELOP branches so that there is no confusion for which version these tests were designated for. Indication is done by pointing to the specific tag (MASTER) or commit (DEVELOP) of the X-Road source code.
+	
+	
+
+Fork repository:
+- `master` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests
+	- Tagged releases of X-road-tests
+- `develop` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests 
+	- Country branch for test development
+	- Latest country test development source code 
+	- Make pull request to develop for ria-ee/X-Road-tests
```

Comment:
I think the purpose of the fork's branches is already clear (the same as in the master repository). Instead we could mention the [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) as the way the repositories are arranged.

Reply:
Changed bit, but still discussed that will be leaved to document as more clarification.